### ISSUE TITLE
Add option for external Postgres instance

### ISSUE THREAD

**Issue by rustprooflabs** [rustprooflabs/pgosm-flex#194]
# Details

#### :warning: Blocked by #193 :warning:

Allow specifying a Postgres connection external to the Docker container.  Initial proof of concept from @jacopofar in #186 set the stage but ran into tech debt issues explained in https://github.com/rustprooflabs/pgosm-flex/pull/186#issuecomment-954366047. To handle the issues in that comment the logic within `docker/pgosm_flex.py` needs to be split out and restructured, my hope is that resolving #193 will make adding this feature easier and cleaner.

> Note - The other part of #186 is tracked in #192 .

**Comment by rustprooflabs** [comment]
Supported merged to `dev` in #217.  Can be tested with `dev` image.

```
docker pull rustprooflabs/pgosm-flex:dev
```

**Comment by rustprooflabs** [comment]
Merged into `main` and `:latest` Docker image.  `dev` image removed.
